modules/stanford_soe_helper_homepage/stanford_soe_helper_homepage.module

```diff
@@ -5,3 +5,24 @@
  */
 
 include_once 'stanford_soe_helper_homepage.features.inc';
+
+/**
+ * Implementation of hook_init().
+ */
+function stanford_soe_helper_homepage_init() {
```

Comment:
@boznik I'm thinking that hook_init() is the wrong function here as it gets executed multiple times. What do you suggest?